### ISSUE TITLE
Exception during processing: No such kernel

### ISSUE THREAD

**Issue by destepa** [aws-samples/sagemaker-run-notebook#4]
I´m using the JupyterLab extension in Sagemaker Studio. I´ve installed succesfully the extension, and I´m able to run or schedule a notebook. However, the output notebook is created but the cells are not executed, due to a mismatch between the kernel that is specified in the .ipynb and the kernel that is supposed to be used inside the docker container. 

For instance, if I´ve executed and saved the notebook in Sagemaker Studio with the kernel "Python 3 (Data Science)", the .ipynb metadata  is:

```
...
 "metadata": {
  "instance_type": "ml.t3.medium",
  "kernelspec": {
   "display_name": "Python 3 (Data Science)",
   "language": "python",
   "name": "python3__SAGEMAKER_INTERNAL__arn:aws:sagemaker:eu-west-1:470317259841:image/datascience-1.0"
  }
...
```

Running the notebook with the JupyterLab extension won´t result in a valid output notebook, and the CloudWatch log for the processing job show this error:

`Exception during processing: No such kernel named python3__SAGEMAKER_INTERNAL__arn:aws:sagemaker:eu-west-1:470317259841:image/datascience-1.0
`

This problem is solved if I manually change the name of the kernel in the .ipynb to python3, which is the kernel name specified in the execute.py script used to create the container. The .ipynb will look now as follows:

```
...
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (Data Science)",
   "language": "python",
   "name": "python3"
  }
...
```

This can´t be a proper solution for a production environment, where multiple notebooks are to be created and automated, so I must be missing something in the configuration of the container or the extension so that the kernel name is not changed manually. How can I solve this issue?

**Comment by tomfaulhaber** [comment]
Confirmed. This is a regression in v0.14.0.

I'll try to get this fixed in the next day or so. In the meantime, try installing v0.13.0 and rebuilding your notebook container (with `sagemaker-run-notebook create-container`) to get around this.

Sorry for the inconvenience.

**Comment by destepa** [comment]
Thank you @tomfaulhaber , installing v0.13.0 and rebuilding works.

**Comment by tomfaulhaber** [comment]
Fixed in [Release v0.15.0][release].

[release]: https://github.com/aws-samples/sagemaker-run-notebook/releases/tag/v0.15.0
